.github/workflows/ci.yml

```diff
@@ -52,5 +52,10 @@ jobs:
       - name: Build the packages
         run: yarn build
 
+      - name: Test (Linux)
+        run: xvfb-run -a yarn test
```

Comment:
This is necessary for the VS Code tests, see [this page](https://code.visualstudio.com/api/working-with-extensions/continuous-integration#github-actions)